src/_guides/language/analysis-options.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: guide
+title: "Customize Static Analysis"
+description: "Customize static analysis using the analysis_options file."
+---
+
+Static analysis allows you to find problems before
+executing a single line of code. It's a powerful tool
+used to prevent bugs and ensure that code conforms to style
+guidelines. In the Dart ecosystem,
+the Dart Analysis Server and other tools use the
+[analyzer package](https://pub.dartlang.org/packages/analyzer)
+to perform static analysis.
+
+You can customize static analysis to look for a variety of potential
+problems, including errors and warnings specified in the
+[Dart language spec](/guides/language/spec).
+You can also configure the linter, one of the analyzer's plugins,
+to ensure that your code complies with the
+[Dart Style Guide](/guides/language/effective-dart/style)
+and other suggested guidelines
+in [Effective Dart](/guides/language/effective-dart).
+Dart tools such as the
+[Dart Dev Compiler (DDC)](https://github.com/dart-lang/dev_compiler),
+[`dartanalyzer`](https://github.com/dart-lang/sdk/tree/master/pkg/analyzer_cli#dartanalyzer),
+[`flutter analyze`](https://flutter.io/debugging/#the-dart-analyzer),
+and the [IntelliJ IDEs](https://www.dartlang.org/tools/jetbrains-plugin)
+use the analyzer package to evaluate your code.
+
+This document explains how to customize the behavior of the analyzer
+using an `.analysis_options` file. If you want to
+add static analysis to your tool, see the
+[analyzer package](https://pub.dartlang.org/packages/analyzer) docs and the
+[Analysis Server API Specification](https://htmlpreview.github.io/?https://github.com/dart-lang/sdk/blob/master/pkg/analysis_server/doc/api.html).
+
+
+## The analysis options file
+
+The `.analysis_options` (or `.analysis_options.yaml`) file,
+written in the YAML language, is typically placed in the same
+directory as the pubspec file at the top of the package.
+If no file is present, the analyzer walks up the directory
+tree, looking for one.
+
+Here's a sample YAML analysis_options file:
+
+{% prettify yaml %}
+analyzer:
+  strong-mode:
+    - no-implicit-downcasts: true
+    - no-implicit-dynamic: true
+  errors:
+    todo: ignore
+  exclude:
+    - flutter/**
+    - lib/api/*.dart
+
+linter:
+  rules:
+    - avoid_empty_else
+    - cancel_subscriptions
+    - close_sinks
+{% endprettify %}
+
+YAML is sensitive to whitespace&mdash;don't use tabs in a YAML file,
+and use 2 spaces to denote each level of indentation.
+
+<div class="alert alert-info" markdown="1">
+Note: You might come across a `language:` tag in an analysis options file.
```

Comment:
Didn't we used to use `<aside>` instead of `<div>` for this kind of thing? (Maybe we changed...)